sdk/examples/module/getModulesNumber.ts

```diff
@@ -0,0 +1,5 @@
+import VeraxSdk from "../../src/VeraxSdk";
```

Comment:
should be removed

Reply:
It feels important to demonstrate web3 calls too, no?

Reply:
Moved to `UtilsDataMapper`